File: sub-mysql-web-service/src/main/java/org/github/mysql/web/base/mapper/ISysUserRoleMapper.java
package org.github.mysql.web.base.mapper;

import org.github.mysql.web.base.entity.SysUserRoleEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * <p>
 * 用户与角色对应关系 Mapper 接口
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-22
 */
@MyBatisMapper
public interface ISysUserRoleMapper extends IMapper<SysUserRoleEntity> {}

File: sub-mysql-web-service/src/main/java/org/github/mysql/web/base/mapper/ISysUserRoleMapper.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.web.base.mapper;

import org.github.mysql.web.base.entity.SysUserRoleEntity;
import org.github.base.IMapper;
import org.github.mybatis.MyBatisMapper;

/**
 * 用户与角色对应关系 Mapper 接口
 *
 * @author JYD_XL
 * @since 2021-05-30
 */
@MyBatisMapper
interface ISysUserRoleMapper : IMapper<SysUserRoleEntity>
